forms/jqm/subformcollapse.js

```diff
@@ -35,13 +43,17 @@ define(function (require) {
       this.$summary = this.$collapsible.children('h3')
         .children('.bm-subform__bm-summary');
 
-      this.$collapsible.collapsible();
+      this.$collapsible.collapsible({
+        collapsed: this.model.get('collapsed')
+      });
 
       this.$el.attr(
         'data-record-index',
         this.model.parentElement.attributes.forms.indexOf(this.model)
       );
 
+      this.$toggleTrigger = $('.ui-collapsible-heading-toggle', this.$el);
```

Comment:
it does, because the enhancement that creates any sub forms with the selector i am using has not been run and i cache the result, but i'll use `.children` method so if we change anything and the above is no longer true we wont be caught out